src/allmydata/web/common.py

```diff
@@ -1,45 +1,32 @@
 
```

Comment:
Changes to this module are just moving stuff into common_py3.py so nevow imports don't get in the way.

Reply:
The `nevow` imports in `common.py` look minimal. Could we put those behind an `if PY2` in order to leave the rest of `common.py` intact? That would save us from having to fold (even more of) `common_py3.py` back in later (sez the guy who just did [that](https://github.com/tahoe-lafs/tahoe-lafs/pull/842) for `test.common_py3` 😬 ). Maybe something like:

```python3
if PY2:
    from nevow import appserver
    from nevow.inevow import IRequest

    class MyExceptionHandler(appserver.DefaultExceptionHandler, object):                                                                        
        def simple(self, ctx, text, code=http.BAD_REQUEST):
            pass # full version
        def renderHTTP_exception(self, ctx, f):
            pass # full version

    def get_root():
            pass # full version

elif PY3:

    class NotPortedToPython3(NotImplementedError):
         pass

    class MyExceptionHandler(appserver.DefaultExceptionHandler, object):                                                                        
        def simple(self, ctx, text, code=http.BAD_REQUEST):
            raise NotPortedToPython3
        def renderHTTP_exception(self, ctx, f):
            raise NotPortedToPython3

    def get_root():
            raise NotPortedToPython3
    
else:
    raise heck
```

Reply:
Good idea, done.